src/App/Game/GameLayout.tsx

```diff
@@ -6,12 +6,13 @@ import Typography from '@material-ui/core/Typography';
 interface IGameLayoutProps {
   children?: React.ReactNode;
   gameOver?: string;
+  playAgain?: boolean;
 }
 
 export class GameLayout extends React.Component<IGameLayoutProps, {}> {
   render() {
     if (this.props.gameOver) {
-      return (<GameOver result={this.props.gameOver} />);
+      return (<GameOver result={this.props.gameOver} playAgain={this.props.playAgain} />);
```

Comment:
Booleans are always bad parameters, it is better to pass gameArgs or the game mode

Reply:
Agreed, changed